pom.xml

```diff
@@ -674,5 +674,14 @@
         <javadoc.opts>-Xdoclint:none</javadoc.opts>
       </properties>
     </profile>
+    <profile>
+      <id>ci</id>
+      <properties>
+        <user.home>/home/jenkins</user.home>
+        <fabric8.namespace>ipaas-ci</fabric8.namespace>
+        <java-image>fabric8/s2i-java:2.0</java-image>
```

Comment:
This is not a magic constant but must refer to the image configured in [runtime/pom.xml](https://github.com/iocanel/ipaas-rest/blob/569caf995bc5e04f3b05f493c94302df98890215/runtime/pom.xml#L93) : java-base-image